pkg/clients/openshift.go

```diff
@@ -404,3 +420,57 @@ func (o OpenshiftClient) ListRegistryImages() (images []string, err error) {
 	}
 	return imageList, nil
 }
+
+// GetClusterNetworkPlugin - Get cluster network
+func (o OpenshiftClient) GetClusterNetworkPlugin() (string, error) {
+	net := &networkoapi.ClusterNetwork{}
+	err := o.networkClient.Get().Resource("clusternetworks").Name("default").Do().Into(net)
+	if err != nil {
+		return "", err
+	}
+	return net.PluginName, nil
+}
+
+// GetNetNamespace - Get Net Namespace.
+func (o OpenshiftClient) GetNetNamespace(nsName string) (*networkoapi.NetNamespace, error) {
+	netNamespace := &networkoapi.NetNamespace{}
+	err := o.networkClient.Get().Resource("netnamespaces").Name(nsName).Do().Into(netNamespace)
+	if err != nil {
+		return nil, err
+	}
+	return netNamespace, nil
+}
+
+// JoinNamespacesNetworks - Will take the net namespace to be added to a network,
+// and the namespace ID of that netwok.
```

Comment:
make this more clear that the namespace id is the network that we are joining to.